2023-08-23 15:20:06,602 ERROR t.DatabaseBuilder [main] Cannot create database in folder=database\
java.sql.SQLException: Failed to create database 'database\', see the next exception for details.
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(Unknown Source) ~[derby-10.14.2.0.jar:?]
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(Unknown Source) ~[derby-10.14.2.0.jar:?]
	at org.apache.derby.impl.jdbc.Util.seeNextException(Unknown Source) ~[derby-10.14.2.0.jar:?]
	at org.apache.derby.impl.jdbc.EmbedConnection.createDatabase(Unknown Source) ~[derby-10.14.2.0.jar:?]
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source) ~[derby-10.14.2.0.jar:?]
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source) ~[derby-10.14.2.0.jar:?]
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source) ~[derby-10.14.2.0.jar:?]
	at java.security.AccessController.doPrivileged(Native Method) ~[?:1.8.0_382]
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source) ~[derby-10.14.2.0.jar:?]
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source) ~[derby-10.14.2.0.jar:?]
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source) ~[derby-10.14.2.0.jar:?]
	at org.apache.derby.jdbc.AutoloadedDriver.connect(Unknown Source) ~[derby-10.14.2.0.jar:?]
	at java.sql.DriverManager.getConnection(DriverManager.java:664) ~[?:1.8.0_382]
	at java.sql.DriverManager.getConnection(DriverManager.java:270) ~[?:1.8.0_382]
	at table_database.DatabaseBuilder.create(DatabaseBuilder.java:48) [classes/:?]
	at table_database.Database.connect(Database.java:66) [classes/:?]
	at tse_main.StartUI.launch(StartUI.java:217) [classes/:?]
	at tse_main.StartUI.main(StartUI.java:197) [classes/:?]
Caused by: org.apache.derby.iapi.error.StandardException: Failed to create database 'database\', see the next exception for details.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source) ~[derby-10.14.2.0.jar:?]
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.wrapArgsForTransportAcrossDRDA(Unknown Source) ~[derby-10.14.2.0.jar:?]
	... 18 more
Caused by: org.apache.derby.iapi.error.StandardException: Directory C:\Users\thanos\Desktop\tse-efsa\tse-app\database already exists.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source) ~[derby-10.14.2.0.jar:?]
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source) ~[derby-10.14.2.0.jar:?]
	at org.apache.derby.impl.services.monitor.StorageFactoryService$10.run(Unknown Source) ~[derby-10.14.2.0.jar:?]
	at java.security.AccessController.doPrivileged(Native Method) ~[?:1.8.0_382]
	at org.apache.derby.impl.services.monitor.StorageFactoryService.createServiceRoot(Unknown Source) ~[derby-10.14.2.0.jar:?]
	at org.apache.derby.impl.services.monitor.BaseMonitor.bootService(Unknown Source) ~[derby-10.14.2.0.jar:?]
	at org.apache.derby.impl.services.monitor.BaseMonitor.createPersistentService(Unknown Source) ~[derby-10.14.2.0.jar:?]
	at org.apache.derby.impl.services.monitor.FileMonitor.createPersistentService(Unknown Source) ~[derby-10.14.2.0.jar:?]
	at org.apache.derby.iapi.services.monitor.Monitor.createPersistentService(Unknown Source) ~[derby-10.14.2.0.jar:?]
	at org.apache.derby.impl.jdbc.EmbedConnection$5.run(Unknown Source) ~[derby-10.14.2.0.jar:?]
	at java.security.AccessController.doPrivileged(Native Method) ~[?:1.8.0_382]
	at org.apache.derby.impl.jdbc.EmbedConnection.createPersistentService(Unknown Source) ~[derby-10.14.2.0.jar:?]
	... 15 more
2023-08-23 15:20:06,637 ERROR t.Database [main] Cannot get database property key=DB_VERSION java.sql.SQLException: Database 'database\' not found.
2023-08-23 15:20:06,638 ERROR t.StartUI [main] Old version of the database found
table_database.DatabaseVersionException: No database version found. Cannot check updates.
	at table_database.Database.update(Database.java:80) ~[classes/:?]
	at global_utils.EFSARCL.init(EFSARCL.java:41) ~[classes/:?]
	at tse_main.StartUI.launch(StartUI.java:229) [classes/:?]
	at tse_main.StartUI.main(StartUI.java:197) [classes/:?]
2023-08-23 15:20:12,730 FATAL t.StartUI [main] Generic error occurred
java.lang.NullPointerException: null
	at tse_main.StartUI.ask(StartUI.java:182) ~[classes/:?]
	at tse_main.StartUI.launch(StartUI.java:243) ~[classes/:?]
	at tse_main.StartUI.main(StartUI.java:197) [classes/:?]
